content/_index.md

```diff
@@ -83,6 +83,12 @@ minecraft = "f1bade07a7764245ae92b749078c99e3"
 [[extra.contributors]]
 name = "Expora9"
 minecraft = "8b07d8ad352e4b86b1bc2d2dad269c4b"
+[[extra.contributors]]
```

Comment:
Oops, you did tatertosis!

Reply:
Whats tatertosis?

Reply:
These contributor entries are duplicated :)